src/Helpers/CheckoutHelper.php

```diff
@@ -1093,7 +1093,7 @@ public function refundTransaction($order, $context, $item, $state, &$orderItems
         }
 
         $customFields = $this->getCustomFields($order, $context);
-        $customFields['serviceName']            = $item['transaction_method'];
+        //$customFields['serviceName']            = $item['transaction_method'];
```

Comment:
`$request->setServiceName($serviceName);` should accept a valid service name, some config names in shopware6 are not valid buckaroo payment services, this will brake some payment method refunds

Reply:
@vegimcarkaxhija

Reply:
$serviceName will accept a valid service name, also tested most of the payment methods now are fixed after this change, including in3 which was not working, can you double check

Reply:
@harli91